### ISSUE TITLE
Feature Request: Add a new `tvg-number` field in the M3U

### ISSUE THREAD

**Issue by goyney** [kodi-pvr/pvr.iptvsimple#37]
Currently, this add-on automatically assigns a channel number based on increment.

See line 442 of PVRIptvData.cpp

```
channel.iChannelNumber = iChannelNum++;
```

This is problematic because now channel numbers cannot be synced in Kodi to be what they might actually be. For example, NBCSN is 1152 from my provider, but I cannot go to 1152 in Kodi because this add-on assigns some arbitrary channel number based on when it arrives in the M3U listing.

I propose a new M3U value be added, called "tvg-number" or something similar which takes the integer value of the channel number. Then line 442 could be changed to something like:

```
channel.iChannelNumber = tmpChannel.strTvgNumber;
```

Of course, I know it's not that simple as other things may be affected by that change, but it would be a large improvement for the add-on.

**Comment by perfectcolors** [comment]
This is the only feature I wish this add-on had, otherwise it is perfect.

**Comment by excorp** [comment]
This has been fixed #92 .
